examples/single-source-advanced/README.md

```diff
@@ -116,21 +121,21 @@ EOF
 Create database access secrets in our local secret store.
 
 ```bash
-echo -n "very-secret-hostname" > /tmp/example/local-store/db-host
-echo -n "very-secret-username" > /tmp/example/local-store/db-user
-echo -n "very-secret-password" > /tmp/example/local-store/db-pass
+echo -n "very-secret-hostname" > tmp/example/local-store/db-host
+echo -n "very-secret-username" > tmp/example/local-store/db-user
+echo -n "very-secret-password" > tmp/example/local-store/db-pass
 ```
 
 ### Step 5: Perform sync
 
 Secret synchronization is performed using the CLI by executing the sync plan between source and target secret stores.
 
-#### Database secrets
+#### Synchronize database secrets
 
 To synchronize database secrets from our local to Vault secret store, run:
 
 ```bash
-secret-sync --source "/tmp/example/local-store.yml" --target "/tmp/example/vault-store.yml" --sync "/tmp/example/db-secrets-sync.yml"
+secret-sync --source "tmp/example/local-store.yml" --target "tmp/example/vault-store.yml" --sync "tmp/example/db-secrets-sync.yml"
```

Comment:
lets just break this command into multiple lines (secret-sync \ --source \ --target \ --sync)

Reply:
same for the other secret-sync configs